packages/suite-native-navigation/src/types.tsx

```diff
@@ -0,0 +1,31 @@
+import type { BottomTabScreenProps, BottomTabNavigationProp } from '@react-navigation/bottom-tabs';
+import type { StackScreenProps, StackNavigationProp } from '@react-navigation/stack';
+import { NavigatorScreenParams, ParamListBase } from '@react-navigation/native';
+import { SettingsStackParamList } from '@suite-native/settings';
+import { RouteTabs } from './routes';
+import type { RouteProp } from '@react-navigation/native';
+
+export type RootTabParamList = {
```

Comment:
please move this to routes file, these are types but directly related to routes so it will be better to keep them together

Reply:
moved